### ISSUE TITLE
[openuv] Could not set field value for field 'index'

### ISSUE THREAD

**Issue by miloit** [openhab/openhab-addons#9262]
Hello get this error

```
[re.config.core.internal.ConfigMapper] - Could not set field value for field 'index': Can not set java.lang.String field org.openhab.binding.openuv.internal.config.SafeExposureConfiguration.index to java.math.BigDecimal
java.lang.IllegalArgumentException: Can not set java.lang.String field org.openhab.binding.openuv.internal.config.SafeExposureConfiguration.index to java.math.BigDecimal
	at jdk.internal.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(UnsafeFieldAccessorImpl.java:167) ~[?:?]
	at jdk.internal.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(UnsafeFieldAccessorImpl.java:171) ~[?:?]
	at jdk.internal.reflect.UnsafeObjectFieldAccessorImpl.set(UnsafeObjectFieldAccessorImpl.java:81) ~[?:?]
	at java.lang.reflect.Field.set(Field.java:780) ~[?:?]
	at org.openhab.core.config.core.internal.ConfigMapper.as(ConfigMapper.java:101) [bundleFile:?]
	at org.openhab.core.config.core.Configuration.as(Configuration.java:80) [bundleFile:?]
	at org.openhab.binding.openuv.internal.handler.OpenUVReportHandler.updateChannel(OpenUVReportHandler.java:234) [bundleFile:?]
	at org.openhab.binding.openuv.internal.handler.OpenUVReportHandler.lambda$2(OpenUVReportHandler.java:149) [bundleFile:?]
	at java.util.ArrayList.forEach(ArrayList.java:1541) [?:?]
	at java.util.Collections$UnmodifiableCollection.forEach(Collections.java:1085) [?:?]
	at org.openhab.binding.openuv.internal.handler.OpenUVReportHandler.updateChannels(OpenUVReportHandler.java:148) [bundleFile:?]
	at org.openhab.binding.openuv.internal.handler.OpenUVReportHandler.lambda$1(OpenUVReportHandler.java:134) [bundleFile:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:305) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:305) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by miloit** [comment]
New configuration necessary but not mentioned in the readme

**Comment by DerOetzi** [comment]
Readme should be corrected at example section to fit new index-option, which is type string now with values ("I", "II", "III", "IV", "V", "VI").

**Comment by jlaur** [comment]
@clinique - since you have just been in this part of the code, can you assess if this is still a problem?

**Comment by clinique** [comment]
No this is not a problem. DrOetzi answer is correct.

**Comment by clinique** [comment]
@jlaur : documentation is already corrected in v3.3.0 of the binding docs. So this issue can be closed.
